mmv1/third_party/terraform/tests/data_source_google_compute_instance_group_manager_test.go

```diff
@@ -0,0 +1,122 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceGoogleComputeInstanceGroupManager(t *testing.T) {
+	t.Parallel()
+
+	zoneName := "us-central1-a"
+	igmName := "tf-tst-igm" + randString(t, 6)
+
+	context := map[string]interface{}{
+		"zoneName":     zoneName,
+		"igmName":      igmName,
+		"baseName":     "tf-tst-igm-base" + randString(t, 6),
+		"poolName":     "tf-tst-pool" + randString(t, 6),
+		"templateName": "tf-tst-templt" + randString(t, 6),
+		"autoHealName": "tf-tst-ah-name" + randString(t, 6),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleComputeInstanceGroupManager_basic(context),
```

Comment:
Can we add another test step to confirm that using the zone/name works too?

Reply:
ok